tests/example-program-gen/exp/accounts/State.ts

```diff
@@ -170,6 +170,24 @@ export class State {
     return this.decode(info.data)
   }
 
+  static async fetchMultiple(
+    c: Connection,
+    addresses: PublicKey[]
+  ): Promise<Array<State | null>> {
+    const infos = await c.getMultipleAccountsInfo(addresses)
+
+    return infos.map((info) => {
+      if (info === null) {
+        return null
+      }
+      if (!info.owner.equals(PROGRAM_ID)) {
+        throw new Error("account doesn't belong to this program")
```

Comment:
We should probably have more exact error message here but let's do this some other time.

Reply:
There is a question whether you'd actually want to throw this, or the exception thrown in decode when you're doing a bulk fetch, rather than just return null and let the rest of the batch continue. Will leave as it is for now.

Reply:
I think that if you attempt to fetch an account that doesn't belong to a program that the client is generated for, it means that you have a bug in your code (you're passing an account that you shouldn't) and we should throw as early as possible for safety reasons.